### PR TITLE
nodetool statuses after every nemesis

### PR DESCRIPTION
**Pull request by abvgedeika**
print nodetool status(on every node) before and after every nemesis

### CODE REVIEW

**Comment by abvgedeika** [comment]
example of run http://jenkins.cloudius-systems.com:8080/job/scylla-1.7-longevity/label=master/16/console

**Comment by roydahan** [comment]
Please fix the PEP8 issues:

/home/travis/build/scylladb/scylla-cluster-tests/sdcm/nemesis.py:257:31: E128 continuation line under-indented for visual indent
/home/travis/build/scylladb/scylla-cluster-tests/sdcm/nemesis.py:260:31: E128 continuation line under-indented for visual indent

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -178,14 +178,6 @@ def disrupt_nodetool_drain(self):
...
         result = self._run_nodetool(drain_cmd, self.target_node)
-        for node in self.cluster.nodes:
-            if node == self.target_node:
-                self.log.info('Status for target %s: %s', node,
-                              self._run_nodetool('nodetool status', node))
-            else:
-                self.log.info('Status for regular %s: %s', node,
-                              self._run_nodetool('nodetool status', node))
-
         if result is not None:
```
Comment (roydahan) [inline sdcm/nemesis.py]: Let's keep this nodetool status, because this is DURING the disruption vs the BEFORE and AFTER

Comment (amoskong) [inline sdcm/nemesis.py]: Agree, the status checking might found some problem which disappears after restart.

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -250,14 +242,27 @@ def repair_nodetool_rebuild(self):
...
-def log_time_elapsed(method):
+def log_time_elapsed_and_status(method):
     """
     Log time elapsed for method to run
 
     :param method: Remote method to wrap.
     :return: Wrapped method.
     """
+    def print_nodetool_status(self):
+        for node in self.cluster.nodes:
```
Comment (amoskong) [inline sdcm/nemesis.py]: How about check the nodes number here (before and after nemesis)?
The number should not be changed here.

Comment (abvgedeika) [inline sdcm/nemesis.py]: @amoskong 
do you mean to check the output of 'nodetool status'( the number of nodes before  and after nemesis)?
or check len(self.cluster.nodes) before and after?

Comment (roydahan) [inline sdcm/nemesis.py]: @amoskong, that's interesting, do we want to assert if it's different than expected?
(e.g. decommission didn't succeed?)
or maybe just issue an ERROR for now that we will manually monitor till we'll monitor the longevity jobs in a smarter way...

Comment (amoskong) [inline sdcm/nemesis.py]: @abvgedeika both of them.

I touched some error case, new node failed to created in decommission test. len(self.nodes) is right, the node doesn't exist/live.

@roydahan add an assert is enough.

Comment (roydahan) [inline sdcm/nemesis.py]: Yes, but asserting will Fail the entire test.
on one hand, I'm in favor so we can login and start to investigate.
OTOH, we don't have any assert at the moment, and maybe it should be part of a bigger task that will be tested separately.

**Pull request merged**